cypress/component/annot-continuous-values.cy.js

```diff
@@ -24,7 +24,7 @@ const store = {
 
     mutations: {
 
-        designateAsMissing: () => (p_columnName, p_rawValue) => {},
+        changeMissingStatus: () => (p_columnName, p_rawValue, p_markAsMissing) => {},
```

Comment:
Nice catch!
Maybe a comment here mentioning the mutation is missing in the component could prove helpful later.

Reply:
Added a note in my last commit for this.